### PR TITLE
removed feature flag from accordian and invalid date

### PR DESCRIPTION
**Pull request by shubhamjaintech**
Fixes https://github.com/Real-Dev-Squad/website-members/issues/569

### What is the change?
Removal of feature flag from 2 bug fixes related to below merged MR.
https://github.com/Real-Dev-Squad/website-members/pull/561
https://github.com/Real-Dev-Squad/website-members/pull/565

### Is it bug?
No
- Steps to repro
NA

### \*Dev Tested?

- [x] Yes
- [ ] No

#### Platforms

- [x] Web

#### Browsers

- [x] Chrome
- [ ] Safari
- [ ] Firefox

### Before / After Change Screenshots
- Before
It needed a feature flag for below scenario

- After - without need of feature flag.

Scenario 1: Invalid Date
image

Scenario 2: Expanding accordian section with no data)
image

Test results:
image

### CODE REVIEW

File: src/components/member-profile/contribution-type/index.js [hunk 1]
```diff
@@ -17,53 +17,34 @@ const renderContributions = (
...
-      </p>
-    );
+  if (contributions?.length > 0) {
+    return contributions.map((noteWorthyContribution, index) => (
+      <Contribution
+        contribution={noteWorthyContribution}
+        key={index}
+        fullName={fullName}
+        imageLink={imageLink}
+        devUser={devUser}
```
Comment (shubhamsigdar1) [inline src/components/member-profile/contribution-type/index.js:27]: Why devUser prop here??

File: src/components/member-profile/contribution-type/index.js [hunk 2]
```diff
@@ -115,23 +85,16 @@ const ContributionType = (props) => {
...
-                  imageLink,
-                  devUser
-                )}
+            {renderContributions(
+              contributions,
+              fullName,
+              imageLink,
+              devUser,
+              type
+            )}
```
Comment (shubhamsigdar1) [inline src/components/member-profile/contribution-type/index.js:94]: Can you please verify here...As I am seeing previous changes
when devUser true then prop- type used

```
{devUser
              ? renderContributions(
                  contributions,
                  fullName,
                  imageLink,
                  devUser,
                  type
                )
              : renderContributions(
                  contributions,
                  fullName,
                  imageLink,
                  devUser
                )}
                
                ```

File: src/components/member-profile/contribution-type/index.js [hunk 3]
```diff
@@ -115,23 +85,16 @@ const ContributionType = (props) => {
...
-                  contributions,
-                  fullName,
-                  imageLink,
-                  devUser
-                )}
+            {renderContributions(
+              contributions,
+              fullName,
+              imageLink,
+              devUser,
```
Comment (shubhamsigdar1) [inline src/components/member-profile/contribution-type/index.js:92]: Why used devUser prop??

File: src/components/member-profile/contribution-type/index.js (hunk 2 above)
Comment (shubhamjaintech) [reply src/components/member-profile/contribution-type/index.js:94]: No proptype is used in both the condition. PFA below screenshot.

image

File: src/components/member-profile/contribution-type/index.js (hunk 3 above)
Comment (shubhamjaintech) [reply src/components/member-profile/contribution-type/index.js:92]: It was already there before, as some old code still use featureflag in their flow. So we cant remove it as discussed in other comments. Thanks Shubham

File: src/components/member-profile/contribution-type/index.js [hunk 4]
```diff
@@ -115,23 +85,16 @@ const ContributionType = (props) => {
...
+              type
```
Comment (shubhamsigdar1) [inline src/components/member-profile/contribution-type/index.js:93]:
```suggestion
              type
```

**Review by shubhamsigdar1 — APPROVED** [review]

**Review by Achintya-Chatterjee — APPROVED** [review]

**Pull request merged**